cognition_objects/environment_variable.py

```diff
@@ -46,13 +48,36 @@ def get_by_name(
     )
 
 
+def get_by_name_and_org_id(
+    org_id: str,
+    name: str,
+) -> CognitionEnvironmentVariable:
+
+    return (
+        session.query(CognitionEnvironmentVariable)
+        .filter(
+            and_(
+                CognitionEnvironmentVariable.organization_id == org_id,
+                CognitionEnvironmentVariable.project_id == None,
+                CognitionEnvironmentVariable.name == name,
+            )
+        )
+        .first()
+    )
```

Comment:
why do we write the default and_ in this filter? Usually we use it directly:
```suggestion
    return (
        session.query(CognitionEnvironmentVariable)
        .filter(            
                CognitionEnvironmentVariable.organization_id == org_id,
                CognitionEnvironmentVariable.project_id == None,
                CognitionEnvironmentVariable.name == name,            
        )
        .first()
    )
```